hack/tests/subcommand-generate-csv.sh

```diff
@@ -14,30 +14,24 @@ OPERATOR_VERSION="0.0.4"
 OPERATOR_BUNDLE_ROOT_DIR="deploy/olm-catalog/${OPERATOR_NAME}"
 DEFAULT_BUNDLE_DIR="${OPERATOR_BUNDLE_ROOT_DIR}/${OPERATOR_VERSION}"
 OUTPUT_DIR="foo"
-OUTPUT_BUNDLE_DIR="${OUTPUT_DIR}/olm-catalog/${OPERATOR_NAME}/${OPERATOR_VERSION}"
-
-function csv_file_for_dir_legacy() {
-  echo "${1}/${OPERATOR_NAME}.v${OPERATOR_VERSION}.clusterserviceversion.yaml"
-}
+OUTPUT_BUNDLE_ROOT_DIR="${OUTPUT_DIR}/olm-catalog/${OPERATOR_NAME}"
+OUTPUT_BUNDLE_DIR="${OUTPUT_BUNDLE_ROOT_DIR}/${OPERATOR_VERSION}"
 
 function check_csv_file_legacy() {
-  check_file "$1" "$(csv_file_for_dir_legacy "$2")" $3
-}
-
-function csv_file_for_dir() {
-  echo "${1}/${OPERATOR_NAME}.clusterserviceversion.yaml"
+  check_file "$1" "${2}/${OPERATOR_NAME}.v${OPERATOR_VERSION}.clusterserviceversion.yaml" $3
 }
 
 function check_csv_file() {
-  check_file "$1" "$(csv_file_for_dir "$2")" $3
+  check_file "$1" "${2}/${OPERATOR_NAME}.clusterserviceversion.yaml" $3
 }
 
-function crd_files_for_dir() {
-  echo "${1}/cache.example.com_memcacheds_crd.yaml ${1}/cache.example.com_memcachedrs_crd.yaml"
+function check_package_file() {
+  check_file "$1" "${2}/${OPERATOR_NAME}.package.yaml" $3
 }
 
 function check_crd_files() {
-  for file in $(crd_files_for_dir "$2"); do check_file "$1" "$file" $3; done
+  local crd_files="${2}/cache.example.com_memcacheds_crd.yaml ${2}/cache.example.com_memcachedrs_crd.yaml"
```

Comment:
Nit: thoughts on defining vars for `cache.example.com_memcacheds_crd.yaml` and `cache.example.com_memcachedrs_crd.yaml`?
It's kind of tied to our tests manifests but that might highlight it more clearly.